src/main/java/net/neoforged/neoforge/client/ClientNeoForgeMod.java

```diff
@@ -157,4 +161,13 @@ public ResourceLocation getFlowingTexture() {
             }
         }, milkType));
     }
+
+    @SubscribeEvent(priority = EventPriority.LOWEST)
+    static void registerSpawnEggColors(RegisterColorHandlersEvent.Item event) {
+        SpawnEggItem.eggs().forEach(egg -> {
```

Comment:
How does a dev with their own coloring be able to override/skip this default behavior?

Reply:
By registering a handler first or returning -1 in `getColor`.